assets/src/edit-story/elements/text/edit.js

```diff
@@ -85,6 +86,14 @@ const TextBox = styled.div`
   right: 0;
 `;
 
+const Highlight = styled.span`
+  ${({ highlightColor }) =>
+    convertToCSS(generatePatternStyles(highlightColor))};
```

Comment:
You actually don't need `convertToCSS` here - if the inline function returns an object, it is assumed to be a valid CSS object and automatically converted. Quite magic!

Reply:
Right, I just blindly copied it from another place, fixed there too https://github.com/google/web-stories-wp/pull/3184/commits/f7505468f3e1eeedbd2b5c205152140bcd0ebd44